e2e/utils/command-utils.ts

```diff
@@ -272,25 +280,33 @@ export function runNgAdd(
   try {
     const pmc = getPackageManagerCommand();
     packageInstall(packageName, undefined, version);
-    return execSync(pmc.run(`ng g ${packageName}:ng-add`, command ?? ''), {
+    const fullCommand = pmc.run(
+      `ng g ${packageName}:ng-add`,
+      `${command}${isVerbose() ? ' --verbose' : ''}` ?? ''
+    );
+    const result = execSync(fullCommand, {
       cwd: tmpProjPath(),
       stdio: 'pipe',
       env: { ...(opts.env || getStrippedEnvironmentVariables()) },
       encoding: 'utf-8',
-    })
-      .toString()
-      .replace(
-        /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g,
-        ''
-      );
+    })?.replace(
+      /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g,
```

Comment:
we can use the stripConsoleColors here to keep it a a bit dry.

Reply:
Thanks!! :D Done! Can you check again please?